.github/workflows/rust.yml

```diff
@@ -133,6 +133,9 @@ jobs:
       - name: Checkout source
         uses: actions/checkout@v4
 
+      - name: Install LLVM
```

Comment:
Can we also modify the "Reclaim disk space" step so that it succeeds? (and hence installing LLVM does make some sense)